.github/workflows/test_and_deploy.yml

```diff
@@ -31,7 +31,10 @@ jobs:
       - name: Check out repository
         uses: actions/checkout@v2
 
-      # https://github.com/ruby/setup-ruby
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16'
```

Comment:
Should we still commit this? There seems to be some version of node installed, but I guess it is better to be explicit about the version. What do you think?

Reply:
Node 16 is the LTS release right now: https://nodejs.org/en/about/releases/
My local installation is on 17 already though. Pinning the version makes it explicit.